src/components/navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FaUserCircle } from 'react-icons/fa';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Books',
+    },
+    {
+      id: 2,
+      path: '/categories',
+      text: 'Categories',
+    },
+  ];
+
+  return (
+    <nav className="Navbar">
+      <div className="Nav1">
+        <header>
+          <h1>Bookstore CMS</h1>
+        </header>
+        <ul className="Links">
+          {links.map((link) => (
+            <li key={link.id}>
+              <NavLink to={link.path} exact>
+                {link.text}
+              </NavLink>
```

Comment:
- [ ] Great job iterating over to render the links more efficiently, but please use the `<Link>` component instead of the `<NavLink>` one in order to comply with the project requirements.